packages/typo3-docs-theme/assets/sass/components/_button.scss

```diff
@@ -2,6 +2,12 @@
     display: inline-flex;
     align-items: center;
     text-align: left;
+    &:focus-visible {
+        // Override the default btn styles from bootstrap as they are poorly visible
+        @include focus();
+        outline: auto;
```

Comment:
This should not be necessary, as `outline: auto` is a shortcut for `outline-color: auto` which is overwritten below.
https://developer.mozilla.org/en-US/docs/Web/CSS/outline#formal_definition

Reply:
If you don't do it the box-shadow is not visible.